tests/E2E/Dockerfile

```diff
@@ -8,18 +8,21 @@ ENV PS_FOLDER_ADMIN admin-dev
 ENV PS_FOLDER_INSTALL install-dev
 ENV DB_SERVER localhost
 ENV TRAVIS_BUILD_DIR /var/www/html
+ENV DB_USER macfly
+ENV TEST_CAMPAIGN full-test
 
 # MySQL installation
 ENV DEBIAN_FRONTEND noninteractive
 RUN echo mysql-server-5.6 mysql-server/root_password password $DB_PASSWD | debconf-set-selections
 RUN echo mysql-server-5.6 mysql-server/root_password_again password $DB_PASSWD | debconf-set-selections
-RUN curl -sS https://downloads.mariadb.com/MariaDB/mariadb_repo_setup | bash -s -- --mariadb-server-version=mariadb-10.1 \
-	&& apt install -y mariadb-server
+RUN apt update \
+	&& apt install -y mysql-server
 # MySQL configuration
-RUN sed -i -e"s/^bind-address\s*=\s*127.0.0.1/bind-address = 0.0.0.0/" /etc/mysql/my.cnf
+RUN sed -i -e"s/^bind-address\s*=\s*127.0.0.1/bind-address = 0.0.0.0/" /etc/mysql/mariadb.conf.d/50-server.cnf
 
 RUN service mysql start \
-	&& mysqladmin -h $DB_SERVER -P $DB_PORT -u $DB_USER -p$DB_PASSWD create $DB_NAME --force
+        && mysql -e "CREATE USER '$DB_USER'@'localhost' IDENTIFIED BY '$DB_PASSWD'; GRANT ALL PRIVILEGES ON *.* TO '$DB_USER'@'localhost'; FLUSH PRIVILEGES;" mysql \
+        && mysqladmin -h $DB_SERVER -P $DB_PORT -u $DB_USER -p$DB_PASSWD create $DB_NAME --force
 
 RUN sed -i -e"s/^exec\s*apache2-foreground/#exec apache2-foreground/" /tmp/docker_run.sh
 RUN service mysql start && /tmp/docker_run_git.sh
```

Comment:
`service mysql start` is ran on line 23, should be a `restart` for the first one I think, and second is useless, no need to restart mysql after adding permissions or user.

Reply:
In some case the service is stopped and needs to be started again (for instance if you reuse a cached layer to regenerate this one).

Reply:
Weird, really weird ^^